src/include/daos_errno.h

```diff
@@ -246,7 +246,9 @@ extern "C" {
 	       Agent is incompatible with libdaos)			\
 	/** Multiple shards locate on the same target */		\
 	ACTION(DER_SHARDS_OVERLAP,	(DER_ERR_DAOS_BASE + 30),	\
-	       Shards overlap)
+	       Shards overlap)						\
+	ACTION(DER_DF_IMCOMPAT,		(DER_ERR_DAOS_BASE + 31),	\
```

Comment:
I actually prefer this spelling but line 227 is the same error

Reply:
ah indeed